src/applications/mhv/medications/sass/medications.scss

```diff
@@ -325,4 +314,28 @@ va-checkbox.select-all-checkbox::part(label) {
 
 .hide-visited-link:visited {
   color: var(--vads-color-link) !important;
-}
\ No newline at end of file
+}
+
+.va-breadcrumbs-li {
+  padding: 5px 0 !important;
+}
+
+.include-back-arrow {
```

Comment:
is it possible to use the `<va-icon/>` component instead of the css? something like: 

`        <va-icon
          icon="arrow_back"
          size={3}
          srtext="breadcrumb back arrow"
        />`